src/api/CreateChecklistItems.ts

```diff
@@ -0,0 +1,113 @@
+import { spWebContext } from "providers/SPWebContext";
+import { IInRequest } from "api/RequestApi";
+import { EMPTYPES } from "constants/EmpTypes";
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+
+const createInboundChecklistItems = (request: IInRequest) => {
+  const [batchedSP, execute] = spWebContext.batched();
+
+  const checklistItems = batchedSP.web.lists.getByTitle("CheckListItems");
+
+  // Welcome Package
+  // Required for all inbounds?
+  checklistItems.items.add({
+    Title: "Welcome Package",
+    Description: "",
+    Lead: "Supervisor",
+    RequestId: request.Id,
+  });
+
+  // IA Training
+  // Required for all inbounds?
+  checklistItems.items.add({
+    Title: "IA Training",
+    Description: "",
+    Lead: "Employee",
+    RequestId: request.Id,
+  });
+
+  // Transfer CAC if civilian employee already has one
+  if (
+    request.hasExistingCAC === "yes" &&
+    request.empType === EMPTYPES.Civilian
+  ) {
+    checklistItems.items.add({
+      Title: "Transfer CAC",
+      Description: "",
+      Lead: "Security",
+      RequestId: request.Id,
+    });
+  }
+
+  // Obtain CAC if employee doesn't already have one
+  if (request.hasExistingCAC === "no") {
```

Comment:
Same with this -- it should be no for Civ/Mil since they don't get asked the question.